### ISSUE TITLE
Speck integration tests fail with timeout exceptions.

### ISSUE THREAD

**Issue by mkcor** [plotly/dash-bio#481]
I increased the timeout value to give it a try: https://github.com/plotly/dash-bio/commit/9b6fc7bb36cb412a350bc8ee1fa31681fdac2e0d but `test_dbsp002_click_and_drag(dash_duo)` still fails (and `test_dbsp001_rotate(dash_duo)` fails the same way).

Still this timeout error with `tests/integration/test_speck.py`... :thinking:

_Originally posted by @mkcor in https://github.com/plotly/dash-bio/pull/478#issuecomment-583849790_

**Comment by shammamah** [comment]
@mkcor Could you please try running those specific tests locally? It might shed some insight on what is going on (since you'll be able to see the actual app).

**Comment by mkcor** [comment]
Sure. Locally, they run successfully for me. Troubleshooting now...

**Comment by mkcor** [comment]
@shammamah so the CI runs time out for the first two Speck tests, which don't involve interactions via "View" controls, but involve instead 'direct' interactions on the canvas (clicking to rotate or move around the 3D molecule).

I have inspected both the demo app https://dash-gallery.plotly.host/dash-speck/ and the simple test app locally. The 'canvas' element lives inside of a 'div#(test-)speck' element (geometrically, they have the same size), which is what Selenium fails to wait for (i.e., when the timeout occurs):
https://github.com/plotly/dash-bio/blob/2046d6583984398a1e02c3f9ca00a5ceff890945/tests/integration/test_speck.py#L34
and, equivalently:
https://github.com/plotly/dash-bio/blob/2046d6583984398a1e02c3f9ca00a5ceff890945/tests/integration/test_speck.py#L56

Then, Selenium would normally find the 'canvas' element:
https://github.com/plotly/dash-bio/blob/2046d6583984398a1e02c3f9ca00a5ceff890945/tests/integration/test_speck.py#L36

First, I wondered whether waiting for the embedding element was really necessary, so I tried https://github.com/plotly/dash-bio/commit/06764dcbe1b2754997214182292a87203cccd266 (passed locally, but the CI run couldn't find the canvas element then).

I tried editing the 'tag' (what's the proper name?) for the embedding element https://github.com/plotly/dash-bio/commit/0b6e1fa1e8f618731e9a08b01252ae9f01a1fb72 to no avail either...

I'm confused, because the other four Speck tests also wait for element `'#' + _COMPONENT_ID` via `simple_app_callback()` calls...

**Comment by mkcor** [comment]
Ok, this f16687bc9947c4152f3a1a86ccd5f8858dc6c202 is consistent / expected...

**Comment by shammamah** [comment]
@mkcor Good news! Turns out that the problem is *not* with our tests, but actually with the docker image. I fixed it in https://github.com/plotly/dash-bio/pull/484/commits/0d026a4c9d4cd228b172f36dda02449231a84f4c -- once that is merged into `master`, could you update your branch and see if the tests start passing?

**Comment by mkcor** [comment]
Oh dear, what a relief! It didn't make any sense that selector `#test-speck` couldn't be found for the first two integration tests... Sure, I'll update my branches and ping you for review/approval! Thanks.

**Comment by mkcor** [comment]
I can now close this issue and nuke branch `increase-timeout`.
